app.py

```diff
@@ -658,16 +668,20 @@ def _get_from_cache(url, headers, json=None, method=None):
 
 # Publisher views
 # ===
-@app.route('/snaps/<snap_name>/measure')
-def publisher_snap_measure(snap_name):
+@app.route('/account/snaps/<snap_name>/measure/')
+def publisher_snap(snap_name):
```

Comment:
We are not very consistent in naming these handlers.
As this is a measure page not 'snap page', name could be more specific.

On the other hand for market page we have separate get_ and post_ handlers which makes naming even more inconsistent...

Reply:
This specific case was my bad, have fixed it. But yeah we need to figure out how we're going to make the monolith of a file more manageable.